apps/tasks/web/index.html

```diff
@@ -55,5 +60,6 @@
       });
     });
   </script>
-</body>
-</html>
+
+
+</body></html>
```

Comment:
Weird formatting, do we really need flutter web?

Reply:
it was generated at the initialization, therefore not part of this issue, but good point

Reply:
Moved to internal chat!